torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -2225,6 +2225,48 @@ RegisterOperators reg2({
 
     DEFINE_INT_OP(aten::gcd, gcd(a, b)),
 
+    DEFINE_GENERIC_OP(
```

Comment:
These aren't binary ops like in `DEFINE_GENERIC_OP`, below we have a macro `DEFINE_UNARY_OP` that would fit these better.

Reply:
Will work on now and update shortly

Reply:
Where is DEFINE_UNARY_OP actually defined/declared?  I am not seeing it locally or in the repo in that file or the project at all?(I only searched the whole project locally)

Reply:
See https://github.com/pytorch/pytorch/blob/6dc445e1a84dc5d093d640de54f038f021d13227/torch/csrc/jit/register_prim_ops.cpp#L1085

Reply:
Thanks, fixed and pushing